### PR TITLE
[docs] fixes to sql-scalar.md

### PR DESCRIPTION
**Pull request by edgar2020**
### Description

Makes changes to `sql-functions.md` page:

- fixes definition in table for `REPEAT` (N is not optional, always demands two arguments)
- fixes definition in table for `LEFT` and `RIGHT` (length is not optional)
- removes comma in intro section 
- changes definition for `TEXTCAT` so that it is not reliant on `CONCAT`'s definition
- Changes function to be `CONCAT(expr[, expr, ...])`
- Changes `POSITION` function to be more neutral (removing needle in haystack analogy)
- changed optional parameters to have comma inside brackets

This PR has:
- [X] been self-reviewed.

### CODE REVIEW

File: docs/querying/sql-scalar.md [hunk 1]
```diff
@@ -87,12 +87,12 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
 
-String functions accept strings, and return a type appropriate to the function.
+String functions accept strings and return a type appropriate to the function.
 
 |Function|Notes|
 |--------|-----|
-|`CONCAT(expr, expr...)`|Concats a list of expressions. Also see the [concatenation operator](sql-operators.md#concatenation-operator).|
-|`TEXTCAT(expr, expr)`|Two argument version of `CONCAT`.|
+|`CONCAT(expr, [expr, ...])`|Concatenates a list of expressions. Also see the [concatenation operator](sql-operators.md#concatenation-operator).|
+|`TEXTCAT(expr, expr)`| Concatenates two expressions.|
```
Comment (asdf2014) [inline docs/querying/sql-scalar.md:95]:
```suggestion
|`TEXTCAT(expr, expr)`|Concatenates two expressions.|
```

File: docs/querying/sql-scalar.md [hunk 2]
```diff
@@ -79,50 +79,50 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
-|`TEXTCAT(expr, expr)`|Two argument version of `CONCAT`.|
+|`CONCAT(expr[, expr, ...])`|Concatenates a list of expressions. Also see the [concatenation operator](sql-operators.md#concatenation-operator).|
+|`TEXTCAT(expr, expr)`|Concatenates two expressions.|
 |`CONTAINS_STRING(expr, str)`|Returns true if the `str` is a substring of `expr`.|
 |`ICONTAINS_STRING(expr, str)`|Returns true if the `str` is a substring of `expr`. The match is case-insensitive.|
 |`DECODE_BASE64_UTF8(expr)`|Decodes a Base64-encoded string into a UTF-8 encoded string.|
-|`LEFT(expr, [length])`|Returns the leftmost length characters from `expr`.|
-|`RIGHT(expr, [length])`|Returns the rightmost length characters from `expr`.|
+|`LEFT(expr, length)`|Returns the leftmost `length` characters from `expr`, where `length` is an integer. |
+|`RIGHT(expr, length)`|Returns the rightmost ``length`` characters from `expr`, where `length` is an integer. |
```
Comment (vtlim) [inline docs/querying/sql-scalar.md:100]:
```suggestion
|`RIGHT(expr, length)`|Returns the rightmost `length` characters from `expr`, where `length` is an integer. |
```

File: docs/querying/sql-scalar.md [hunk 3]
```diff
@@ -79,50 +79,50 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
 |`REGEXP_LIKE(expr, pattern)`|Returns whether `expr` matches regular expression `pattern`. The pattern may match anywhere inside `expr`; if you want to match the entire string instead, use the `^` and `$` markers at the start and end of your pattern. Similar to [`LIKE`](sql-operators.md#logical-operators), but uses regexps instead of LIKE patterns. Especially useful in WHERE clauses.|
 |`REGEXP_REPLACE(expr, pattern, replacement)`|Replaces all occurrences of regular expression `pattern` within `expr` with `replacement`. The replacement string may refer to capture groups using `$1`, `$2`, etc. The pattern may match anywhere inside `expr`; if you want to match the entire string instead, use the `^` and `$` markers at the start and end of your pattern.|
 |`REPLACE(expr, pattern, replacement)`|Replaces pattern with replacement in `expr`, and returns the result.|
-|`REPEAT(expr, [N])`|Repeats `expr` N times.|
+|`REPEAT(expr, N)`|Repeats `expr` N times.|
```
Comment (vtlim) [inline docs/querying/sql-scalar.md:116]:
```suggestion
|`REPEAT(expr, N)`|Repeats `expr` _N_ times.|
```

File: docs/querying/sql-scalar.md [hunk 4]
```diff
@@ -263,9 +263,9 @@ The [DataSketches extension](../development/extensions-core/datasketches-extensi
...
-|`DS_TUPLE_DOUBLES_UNION(expr, ..., [nominalEntries])`|Returns a union of tuple sketches, where each input expression must return a tuple sketch which contains an array of double values as its Summary Object. The values contained in the Summary Objects are summed when combined. If the last value of the array is a numeric literal, Druid assumes that the value is an override parameter for [nominal entries](../development/extensions-core/datasketches-tuple.md).|
+|`DS_TUPLE_DOUBLES_INTERSECT(expr, ...[, nominalEntries])`|Returns an intersection of tuple sketches, where each input expression must return a tuple sketch which contains an array of double values as its Summary Object. The values contained in the Summary Objects are summed when combined. If the last value of the array is a numeric literal, Druid assumes that the value is an override parameter for [nominal entries](../development/extensions-core/datasketches-tuple.md).|
```
Comment (vtlim) [inline docs/querying/sql-scalar.md:266]: Something about this notation seems off to me. Should `...` be in the square brackets too?

File: docs/querying/sql-scalar.md [hunk 5]
```diff
@@ -263,9 +263,9 @@ The [DataSketches extension](../development/extensions-core/datasketches-extensi
...
+|`DS_TUPLE_DOUBLES_NOT(expr, ...[, nominalEntries])`|Returns a set difference of tuple sketches, where each input expression must return a tuple sketch which contains an array of double values as its Summary Object. The values contained in the Summary Object are preserved as is. If the last value of the array is a numeric literal, Druid assumes that the value is an override parameter for [nominal entries](../development/extensions-core/datasketches-tuple.md).|
+|`DS_TUPLE_DOUBLES_UNION(expr, ...[, nominalEntries])`|Returns a union of tuple sketches, where each input expression must return a tuple sketch which contains an array of double values as its Summary Object. The values contained in the Summary Objects are summed when combined. If the last value of the array is a numeric literal, Druid assumes that the value is an override parameter for [nominal entries](../development/extensions-core/datasketches-tuple.md).|
```
Comment (vtlim) [inline docs/querying/sql-scalar.md:268]: For these three lines, we should lowercase "summary object" since it's not a proper noun

Comment (edgar2020) [reply docs/querying/sql-scalar.md]: Will address in later PR for the appropriate section

File: docs/querying/sql-scalar.md (hunk 4 above)
Comment (edgar2020) [reply docs/querying/sql-scalar.md]: Will address in later PR for the appropriate section

File: docs/querying/sql-scalar.md [hunk 6]
```diff
@@ -56,9 +56,9 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
 |`LOG10(expr)`|Logarithm (base 10).|
 |`POWER(expr, power)`|`expr` raised to the power of `power`.|
 |`SQRT(expr)`|Square root.|
-|`TRUNCATE(expr, [digits])`|Truncate `expr` to a specific number of decimal digits. If digits is negative, then this truncates that many places to the left of the decimal point. Digits defaults to zero if not specified.|
-|`TRUNC(expr, [digits])`|Alias for `TRUNCATE`.|
-|`ROUND(expr, [digits])`|`ROUND(x, y)` would return the value of the x rounded to the y decimal places. While x can be an integer or floating-point number, y must be an integer. The type of the return value is specified by that of x. y defaults to 0 if omitted. When y is negative, x is rounded on the left side of the y decimal points. If `expr` evaluates to either `NaN`, `expr` will be converted to 0. If `expr` is infinity, `expr` will be converted to the nearest finite double. |
+|`TRUNCATE(expr[, digits])`|Truncate `expr` to a specific number of decimal digits. If digits is negative, then this truncates that many places to the left of the decimal point. Digits defaults to zero if not specified.|
```
Comment (ektravel) [inline docs/querying/sql-scalar.md:59]:
```suggestion
|`TRUNCATE(expr[, digits])`|Truncate `expr` to a specific number of decimal digits. If `digits` is negative, then this truncates that many places to the left of the decimal point. If not specified, `digits` defaults to zero.|
```

File: docs/querying/sql-scalar.md [hunk 7]
```diff
@@ -56,9 +56,9 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
+|`TRUNC(expr[, digits])`|Alias for `TRUNCATE`.|
+|`ROUND(expr[, digits])`|`ROUND(x, y)` would return the value of the x rounded to the y decimal places. While x can be an integer or floating-point number, y must be an integer. The type of the return value is specified by that of x. y defaults to 0 if omitted. When y is negative, x is rounded on the left side of the y decimal points. If `expr` evaluates to either `NaN`, `expr` will be converted to 0. If `expr` is infinity, `expr` will be converted to the nearest finite double. |
```
Comment (ektravel) [inline docs/querying/sql-scalar.md:61]: The use of both `ROUND(expr[, digits])` and `ROUND(x, y)` is confusing. Pick one and update the description accordingly.

File: docs/querying/sql-scalar.md [hunk 8]
```diff
@@ -79,50 +79,50 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
-|`HUMAN_READABLE_BINARY_BYTE_FORMAT(value, [precision])`| Format a number in human-readable [IEC](https://en.wikipedia.org/wiki/Binary_prefix) format. For example, HUMAN_READABLE_BINARY_BYTE_FORMAT(1048576) returns `1.00 MiB`. `precision` must be in the range of `[0, 3]` (default: 2). |
-|`HUMAN_READABLE_DECIMAL_BYTE_FORMAT(value, [precision])`| Format a number in human-readable [SI](https://en.wikipedia.org/wiki/Binary_prefix) format. HUMAN_READABLE_DECIMAL_BYTE_FORMAT(1048576) returns `1.04 MB`. `precision` must be in the range of `[0, 3]` (default: 2). `precision` must be in the range of `[0, 3]` (default: 2). |
-|`HUMAN_READABLE_DECIMAL_FORMAT(value, [precision])`| Format a number in human-readable SI format. For example, HUMAN_READABLE_DECIMAL_FORMAT(1048576) returns `1.04 M`. `precision` must be in the range of `[0, 3]` (default: 2). |
+|`HUMAN_READABLE_BINARY_BYTE_FORMAT(value[, precision])`| Format a number in human-readable [IEC](https://en.wikipedia.org/wiki/Binary_prefix) format. For example, HUMAN_READABLE_BINARY_BYTE_FORMAT(1048576) returns `1.00 MiB`. `precision` must be in the range of `[0, 3]` (default: 2). |
```
Comment (ektravel) [inline docs/querying/sql-scalar.md:82]:
```suggestion
|`HUMAN_READABLE_BINARY_BYTE_FORMAT(value[, precision])`| Format a number in human-readable [IEC](https://en.wikipedia.org/wiki/Binary_prefix) format. For example, `HUMAN_READABLE_BINARY_BYTE_FORMAT(1048576)` returns `1.00 MiB`. `precision` must be in the range of `[0, 3]`. If not specified,  `precision` defaults to 2. |
```

File: docs/querying/sql-scalar.md [hunk 9]
```diff
@@ -79,50 +79,50 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
+|`HUMAN_READABLE_DECIMAL_BYTE_FORMAT(value[, precision])`| Format a number in human-readable [SI](https://en.wikipedia.org/wiki/Binary_prefix) format. HUMAN_READABLE_DECIMAL_BYTE_FORMAT(1048576) returns `1.04 MB`. `precision` must be in the range of `[0, 3]` (default: 2). `precision` must be in the range of `[0, 3]` (default: 2). |
```
Comment (ektravel) [inline docs/querying/sql-scalar.md:83]:
```suggestion
|`HUMAN_READABLE_DECIMAL_BYTE_FORMAT(value[, precision])`| Format a number in human-readable [SI](https://en.wikipedia.org/wiki/Binary_prefix) format. For example, `HUMAN_READABLE_DECIMAL_BYTE_FORMAT(1048576)` returns `1.04 MB`. `precision` must be in the range of `[0, 3]`. If not specified, `precision` defaults to 2. |
```

File: docs/querying/sql-scalar.md [hunk 10]
```diff
@@ -79,50 +79,50 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
+|`HUMAN_READABLE_DECIMAL_FORMAT(value[, precision])`| Format a number in human-readable SI format. For example, HUMAN_READABLE_DECIMAL_FORMAT(1048576) returns `1.04 M`. `precision` must be in the range of `[0, 3]` (default: 2). |
```
Comment (ektravel) [inline docs/querying/sql-scalar.md:84]:
```suggestion
|`HUMAN_READABLE_DECIMAL_FORMAT(value[, precision])`| Format a number in human-readable [SI](https://en.wikipedia.org/wiki/Binary_prefix) format. For example, `HUMAN_READABLE_DECIMAL_FORMAT(1048576)` returns `1.04 M`. `precision` must be in the range of `[0, 3]`. If not specified, `precision` defaults to 2. |
```

File: docs/querying/sql-scalar.md (hunk 7 above)
Comment (ektravel) [reply docs/querying/sql-scalar.md:61]: Also, avoid using "would".

File: docs/querying/sql-scalar.md [hunk 11]
```diff
@@ -79,50 +79,50 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
+|`LEFT(expr, N)`|Returns the `N` leftmost characters from `expr`, where `N` is an integer. |
+|`RIGHT(expr, N)`|Returns the `N` rightmost characters from `expr`, where `N` is an integer. |
 |`LENGTH(expr)`|Length of `expr` in UTF-16 code units.|
 |`CHAR_LENGTH(expr)`|Alias for `LENGTH`.|
 |`CHARACTER_LENGTH(expr)`|Alias for `LENGTH`.|
 |`STRLEN(expr)`|Alias for `LENGTH`.|
-|`LOOKUP(expr, lookupName, [replaceMissingValueWith])`|Look up `expr` in a registered [query-time lookup table](lookups.md) named `lookupName`. The optional constant `replaceMissingValueWith`, if provided, is returned when the `expr` is null or when the lookup does not contain a value for `expr`.<br /><br />Lookups can also be queried directly using the [`lookup` schema](sql.md#from).|
+|`LOOKUP(expr, lookupName[, replaceMissingValueWith])`|Look up `expr` in a registered [query-time lookup table](lookups.md) named `lookupName`. The optional constant `replaceMissingValueWith`, if provided, is returned when the `expr` is null or when the lookup does not contain a value for `expr`.<br /><br />Lookups can also be queried directly using the [`lookup` schema](sql.md#from).|
```
Comment (ektravel) [inline docs/querying/sql-scalar.md:105]:
```suggestion
|`LOOKUP(expr, lookupName[, replaceMissingValueWith])`|Look up `expr` in a registered [query-time lookup table](lookups.md) named `lookupName`. If provided, the optional constant `replaceMissingValueWith` is returned when the `expr` is null or when the lookup does not contain a value for `expr`.<br /><br />You can query lookups directly using the [`lookup` schema](sql.md#from).|
```
Is there a different verb we can use to describe this function? The first sentence is hard to read/comprehend: "Look up `expr` in a registered query-time lookup table named `lookupName`."

File: docs/querying/sql-scalar.md [hunk 12]
```diff
@@ -79,50 +79,50 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
-|`POSITION(needle IN haystack [FROM fromIndex])`|Returns the index of `needle` within `haystack`, with indexes starting from 1. The search will begin at `fromIndex`, or 1 if `fromIndex` is not specified. If `needle` is not found, returns 0.|
-|`REGEXP_EXTRACT(expr, pattern, [index])`|Apply regular expression `pattern` to `expr` and extract a capture group, or `NULL` if there is no match. If index is unspecified or zero, returns the first substring that matched the pattern. The pattern may match anywhere inside `expr`; if you want to match the entire string instead, use the `^` and `$` markers at the start and end of your pattern. Note: when `druid.generic.useDefaultValueForNull = true`, it is not possible to differentiate an empty-string match from a non-match (both will return `NULL`).|
+|`LPAD(expr, length[, chars])`|Returns a string of `length` from `expr` left-padded with `chars`. If `length` is shorter than the length of `expr`, the result is `expr` which is truncated to `length`. The result will be null if either `expr` or `chars` is null. If `chars` is an empty string, no padding is added, however `expr` may be trimmed if necessary.|
```
Comment (ektravel) [inline docs/querying/sql-scalar.md:108]:
```suggestion
|`LPAD(expr, length[, chars])`|Returns a string of `length` from `expr` left-padded with `chars`. If `length` is shorter than the length of `expr`, the result is `expr` which is truncated to `length`. The result is null if either `expr` or `chars` is null. If `chars` is an empty string, no padding is added; however, `expr` may be trimmed if necessary.|
```

File: docs/querying/sql-scalar.md [hunk 13]
```diff
@@ -79,50 +79,50 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
+|`RPAD(expr, length[, chars])`|Returns a string of `length` from `expr` right-padded with `chars`. If `length` is shorter than the length of `expr`, the result is `expr` which is truncated to `length`. The result will be null if either `expr` or `chars` is null. If `chars` is an empty string, no padding is added, however `expr` may be trimmed if necessary.|
```
Comment (ektravel) [inline docs/querying/sql-scalar.md:109]:
```suggestion
|`RPAD(expr, length[, chars])`|Returns a string of `length` from `expr` right-padded with `chars`. If `length` is shorter than the length of `expr`, the result is `expr` which is truncated to `length`. The result is null if either `expr` or `chars` is null. If `chars` is an empty string, no padding is added; however, `expr` may be trimmed if necessary.|
```

File: docs/querying/sql-scalar.md [hunk 14]
```diff
@@ -79,50 +79,50 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
+|`PARSE_LONG(string[, radix])`|Parses a string into a long (BIGINT) with the given radix, or 10 (decimal) if a radix is not provided.|
```
Comment (ektravel) [inline docs/querying/sql-scalar.md:110]:
```suggestion
|`PARSE_LONG(string[, radix])`|Parses a string into a long (BIGINT) with the given radix or 10 (decimal) if a radix is not provided.|
```

File: docs/querying/sql-scalar.md [hunk 15]
```diff
@@ -79,50 +79,50 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
+|`POSITION(substring IN expr [FROM startingIndex])`|Returns the index of `substring` within `expr`, with indexes starting from 1. The search begins at `startingIndex`, if `startingIndex` is not specified, the default is 1. If `substring` is not found, returns 0.|
```
Comment (ektravel) [inline docs/querying/sql-scalar.md:111]:
```suggestion
|`POSITION(substring IN expr [FROM startingIndex])`|Returns the index of `substring` within `expr` with indexes starting from 1. The search begins at `startingIndex`. If `startingIndex` is not specified, the default is 1. If `substring` is not found, returns 0.|
```

File: docs/querying/sql-scalar.md [hunk 16]
```diff
@@ -79,50 +79,50 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
+|`REGEXP_EXTRACT(expr, pattern[, index])`|Apply regular expression `pattern` to `expr` and extract a capture group, or `NULL` if there is no match. If index is unspecified or zero, returns the first substring that matched the pattern. The pattern may match anywhere inside `expr`; if you want to match the entire string instead, use the `^` and `$` markers at the start and end of your pattern. Note: when `druid.generic.useDefaultValueForNull = true`, it is not possible to differentiate an empty-string match from a non-match (both will return `NULL`).|
```
Comment (ektravel) [inline docs/querying/sql-scalar.md:112]:
```suggestion
|`REGEXP_EXTRACT(expr, pattern[, index])`|Apply regular expression `pattern` to `expr` and extract a capture group or `NULL` if there is no match. If `index` is unspecified or zero, returns the first substring that matched the pattern. The pattern may match anywhere inside `expr`. To match the entire string, use the `^` and `$` markers at the start and end of your pattern. When `druid.generic.useDefaultValueForNull = true`, it is not possible to differentiate an empty-string match from a non-match (both return `NULL`).|
```

File: docs/querying/sql-scalar.md [hunk 17]
```diff
@@ -79,50 +79,50 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
 |`REGEXP_LIKE(expr, pattern)`|Returns whether `expr` matches regular expression `pattern`. The pattern may match anywhere inside `expr`; if you want to match the entire string instead, use the `^` and `$` markers at the start and end of your pattern. Similar to [`LIKE`](sql-operators.md#logical-operators), but uses regexps instead of LIKE patterns. Especially useful in WHERE clauses.|
 |`REGEXP_REPLACE(expr, pattern, replacement)`|Replaces all occurrences of regular expression `pattern` within `expr` with `replacement`. The replacement string may refer to capture groups using `$1`, `$2`, etc. The pattern may match anywhere inside `expr`; if you want to match the entire string instead, use the `^` and `$` markers at the start and end of your pattern.|
 |`REPLACE(expr, pattern, replacement)`|Replaces pattern with replacement in `expr`, and returns the result.|
-|`REPEAT(expr, [N])`|Repeats `expr` N times.|
+|`REPEAT(expr, N)`|Repeats `expr` _N_ times.|
```
Comment (ektravel) [inline docs/querying/sql-scalar.md:116]:
```suggestion
|`REPEAT(expr, N)`|Repeats `expr` `N` times.|
```

File: docs/querying/sql-scalar.md [hunk 18]
```diff
@@ -79,50 +79,50 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
...
 |`REVERSE(expr)`|Reverses `expr`.|
-|`STRING_FORMAT(pattern, [args...])`|Returns a string formatted in the manner of Java's [String.format](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#format-java.lang.String-java.lang.Object...-).|
+|`STRING_FORMAT(pattern[, args...])`|Returns a string formatted in the manner of Java's [String.format](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#format-java.lang.String-java.lang.Object...-).|
 |`STRPOS(haystack, needle)`|Returns the index of `needle` within `haystack`, with indexes starting from 1. If `needle` is not found, returns 0.|
-|`SUBSTRING(expr, index, [length])`|Returns a substring of `expr` starting at index, with a max length, both measured in UTF-16 code units.|
-|`SUBSTR(expr, index, [length])`|Alias for `SUBSTRING`.|
+|`SUBSTRING(expr, index[, length])`|Returns a substring of `expr` starting at index, with a max length, both measured in UTF-16 code units.|
```
Comment (ektravel) [inline docs/querying/sql-scalar.md:120]:
```suggestion
|`SUBSTRING(expr, index[, length])`|Returns a substring of `expr` starting at `index`, with a max `length`, both measured in UTF-16 code units.|
```

**Review by ektravel — COMMENTED** [review]
Left some suggestions.

File: docs/querying/sql-scalar.md (hunk 11 above)
Comment (edgar2020) [reply docs/querying/sql-scalar.md:105]: Have not had the chance to work with LOOKUP in this context, will get back to you if I can think of a better verb

File: docs/querying/sql-scalar.md (hunk 17 above)
Comment (edgar2020) [reply docs/querying/sql-scalar.md:116]: Will change to `N` but note @vtlim suggested `_N_` in this comment: https://github.com/apache/druid/pull/16826#discussion_r1700791136

Style guide: "Use italics to draw attention to a specific word such as when defining terms. Do so sparingly and only when necessary." N is not being defined so I agree with using `N` instead

**Review by ektravel — APPROVED** [review]
The changes look good. `ROUND` and `LOOKUP` can be addressed on a separate PR.

**Review by ektravel — APPROVED** [review]
Changes to the `ROUND` and `LOOKUP` functions look good.

**Review by vtlim — APPROVED** [review]

**Pull request merged**